### PR TITLE
Fix random item

### PR DESCRIPTION
**Pull request by LucianoPAlmeida**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [ ] New extensions are written in Swift 4.
- [ ] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [ ] I have added tests for new extensions, and they passed.
- [ ] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [ ] All extensions are declared as **public**.
- [x] I have added a changelog entry describing my changes.

### CODE REVIEW

**Comment by LucianoPAlmeida** [comment]
Fixing #404 :) 
Let me know if you can't find this issue 😂 
Sorry couldn't resist the bad joke 😂😂😂

File: Sources/Extensions/SwiftStdlib/CollectionExtensions.swift [hunk 1]
```diff
@@ -52,7 +52,8 @@ public extension Collection {
 public extension Collection where Index == Int, IndexDistance == Int {
 	
 	/// SwifterSwift: Random item from array.
-	public var randomItem: Iterator.Element {
+	public var randomItem: Iterator.Element? {
+        guard !isEmpty else { return nil }
```
Comment (akuzminskyi) [inline Sources/Extensions/SwiftStdlib/CollectionExtensions.swift]: Could you please remove a space before **guard**

Comment (LucianoPAlmeida) [reply Sources/Extensions/SwiftStdlib/CollectionExtensions.swift]: hey @akuzminskyi Sorry I don't get it 😅 What space?

Comment (SD10) [reply Sources/Extensions/SwiftStdlib/CollectionExtensions.swift]: The line is a git addition so it looks offset from the `+`

Comment (SD10) [reply Sources/Extensions/SwiftStdlib/CollectionExtensions.swift]: @LucianoPAlmeida Do we still need the `Iterator` prefix on `Element`?

Comment (LucianoPAlmeida) [reply Sources/Extensions/SwiftStdlib/CollectionExtensions.swift]: @SD10 No, I think we can remove it :)

**Pull request merged**